rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -361,7 +361,7 @@ public List<UserJobData> jobsInfoListPost(String sessionId, List<String> jobsId)
     @Override
     public RestMapPage<Long, ArrayList<UserJobData>> revisionAndJobsInfo(String sessionId, int index, int limit,
             boolean myJobs, boolean pending, boolean running, boolean finished, boolean childJobs, String jobName,
-            String projectName, String tenant, String userName, Long parentId, String sortParams) throws RestException {
+            String projectName, String userName, String tenant, Long parentId, String sortParams) throws RestException {
```

Comment:
@fviale what is the tenant ?

Reply:
@amouhoub it was shown at the last friday demo